test/private/initializeGlobalState.m

```diff
@@ -0,0 +1,47 @@
+function orig = initializeGlobalState()
+% Initialize global state. Set various properties of the graphical root to ensure reliable output of the ACID testsuite.
+% See #542 and #552
+    fprintf('Initialize global state...\n');
+
+    %--- Define desired global state properties
+    % See http://undocumentedmatlab.com/blog/getundoc-get-undocumented-object-properties
+    new.defaultAxesColorOrder.val   =               ...
```

Comment:
Thanks for elaborating -- I will adapt and update the hashes (no problem, since I currently still have the versions at my disposal).

I actually took a stab at this issue, to prevent the blind dumping of `get(0)`, as was started off somewhere else.